pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -628,9 +628,9 @@ message Image {
     // ID of the image.
     optional string id = 1;
     // Other names by which this image is known.
-    repeated string repo_tags = 2;
+    repeated string names = 2;
```

Comment:
Are you asking what's the value of these two when someone specify the image name as digest when creating the container?

IIUC, for the return value of `docker inspect`, the names are in the format of `ubuntu:latest`, digests are `ubuntu@sha256:45b23dee08af5e43a7fea6c4cf9c25ccf269ee113168c19722f87876677c5cb2`

Both digests and names can be used to reference and pull the image. Digests also act as a content hash of the image.